src/aap_eda/core/validators.py

```diff
@@ -57,19 +60,95 @@ def check_if_de_exists(decision_environment_id: int) -> int:
 
 
 def check_if_de_valid(image_url: str, eda_credential_id: int):
+    # The OCI standard format for the image url is a combination of a host
+    # (with optional port) separated from the image path (with optional tag) by
+    # a slash: <host>[:port]/<path>[:tag].
+    #
+    # https://github.com/opencontainers/distribution-spec/blob/8376368dd8aadc33bf6c88a8b765df90287bb5c8/spec.md?plain=1#L155 # noqa: E501
+    #
+    # We split the image url on the first slash into the host and path.  The
+    # path is further split into a name and tag on the rightmost colon.
+    #
+    # THe host portion is validated using the validators package and the path
+    # and tag are validated using the OCI regexes for each.
+    base_message = f"Image url {image_url} is malformed; "
```

Comment:
@jshimkus-rh When using gettext  we should be using %s instead of f strings for messages. That data goes to the translators.

Reply:
@mkanoor updated for get text